x-pack/test/alerting_api_integration/common/lib/object_remover.ts

```diff
@@ -41,7 +41,7 @@ export class ObjectRemover {
             `${getUrlPrefix(spaceId)}/${isInternal ? 'internal' : 'api'}/${plugin}/${type}/${id}`
           )
           .set('kbn-xsrf', 'foo')
-          .expect(204);
+          .expect(plugin === 'saved_objects' ? 200 : 204);
```

Comment:
I'm deleting a saved object, and the successful response for this api is [200](https://www.elastic.co/guide/en/kibana/current/saved-objects-api-delete.html#saved-objects-api-delete-response-codes)